library/core/src/slice/raw.rs

```diff
@@ -133,8 +133,7 @@ pub const unsafe fn from_raw_parts_mut<'a, T>(data: *mut T, len: usize) -> &'a m
 
 // In debug builds checks that `data` pointer is aligned and non-null and that slice with given `len` would cover less than half the address space
 #[cfg(debug_assertions)]
-#[unstable(feature = "const_slice_from_raw_parts", issue = "67456")]
-#[rustc_const_unstable(feature = "const_slice_from_raw_parts", issue = "67456")]
+#[rustc_const_stable(since = "1.61.0")]
```

Comment:
Admittedly, this is a change I'm not 100% sure on: its callers are stable in a non-const context, and yet it was marked `unstable`.  I have no real clue what `unstable` even implies for a non-exported function, so this was mostly a no-thinking-involved mechanical change.